response_operations_ui/views/surveys.py

```diff
@@ -34,7 +34,26 @@ def view_survey(short_name):
             "title": f"{survey_details['survey']['surveyRef']} {survey_details['survey']['shortName']}",
         }
     ]
+
+    # Mapping backend states to frontend sates for the user
+    for collection_exercise_state in survey_details["collection_exercises"]:
+        if collection_exercise_state["state"] == "INIT":
```

Comment:
Are any of these likely to change?

Reply:
@emilioward @dcdarrell9 I think @madeye-matt is changing a lot of the state definitions for collection exercises. Are these up to date with the new designs (on Trello)?

Reply:
I mapped these against what I saw in the trello design, but i'm not sure if they were going to change again further, this is something I need to clarify